### ISSUE TITLE
oasis config

### ISSUE THREAD

**Issue by marklrh** [marklrh/otter#10]
I am using ``opam pin`` to test my library and I think there is something wrong with my config. To reproduce the problem, ``opam pin`` branch [oasis_config](https://github.com/marklrh/otter/tree/oasis_config). I fail to compile the following code with command
``ocamlbuild -use-ocamlfind -pkg otter,otter.unix_lwt test.native``
````OCaml
open Otter_unix_lwt

let () = ()
````

**Comment by marklrh** [comment]
also, compiling the following code does not work
````OCaml
open Otter_unix_lwt

module A = Api_unix
module O = Oauth_unix

let () = ()
````

in utop
````OCaml
utop # #require "otter.unix_lwt";;
File "", line 1, characters 0-4:                                                                                                       Error: Parse error: illegal begin of top_phrase                                                                                        Fatal error: exception Exit                                                                                                            
Raised at file "bytes.ml", line 220, characters 25-34
````

**Comment by marklrh** [comment]
closed via #11
